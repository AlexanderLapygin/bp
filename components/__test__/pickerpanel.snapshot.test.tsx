// components/__tests__/pickerpanel.snapshot.test.tsx
import * as React from "react";
import * as TestRenderer from 'react-test-renderer';
import Button from "../PickerPanel";


it('renders correctly', () => {
    const tree = TestRenderer.create(<Button buttonText="Some Text" />).toJSON()
    expect(tree).toMatchSnapshot()
})
