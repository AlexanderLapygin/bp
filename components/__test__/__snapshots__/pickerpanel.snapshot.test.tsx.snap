// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center"
>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <div
      className="MuiBox-root MuiBox-root-105"
      onClick={[Function]}
    >
      cardText1
    </div>
    <div
      className="MuiBox-root MuiBox-root-106"
      onClick={[Function]}
    >
      cardText2
    </div>
  </div>
</div>
`;
